client/src/styles/login-form.scss

```diff
@@ -76,6 +76,10 @@ fieldset {
   padding: 0;
 }
 
+.login-form__help-icon {
```

Comment:
```suggestion
&__help-icon {
```

Reply:
Admittedly, the rest of the SCSS file doesn't follow normal BEM syntax. I'm fine keeping like this. Just want to point out there's a better way.